.env

```
@@ -0,0 +1,40 @@
+Domain=task.labsacme.com
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Domain key should be in uppercase

(LowercaseKey)